test/base/index_range.cpp

```diff
@@ -1,7 +1,9 @@
-// SPDX-FileCopyrightText: 2017 - 2024 The Ginkgo authors
+// SPDX-FileCopyrightText: 2017 - 2025 The Ginkgo authors
 //
 // SPDX-License-Identifier: BSD-3-Clause
 
+/*@GKO_PREPROCESSOR_FILENAME_HELPER@*/
```

Comment:
I am using this weird comment structure to make sure both the original file and the generated file are valid C++ code. The ´#line` directive can start after a comment, but afaik it can't end with a comment, so that needs to be in the next line.